source/documentation/incidents/process.html.md.erb

```diff
@@ -68,18 +57,20 @@ This is whoever discovered the issue and declared the incident.
 
 The incident reporter should call in a designated incident lead, typically a Technical Architect, Lead or Senior WebOps.
 
+The incident lead should be someone from outside the team(s) affected by the incident to provide another set of eyes thoughout the incident lifecycle.
```

Comment:
Maybe something on lead as coordination role and how experience previously running incidents is part of it.

Reply:
Cheers, I've reworded that a bit